tests/vasp/test_vasp_import.py

```diff
@@ -45,10 +45,9 @@ def test_import(self):
         folder_path = os.path.join(
             self.file_location, "../static/vasp_test_files/full_job_minor_glitch"
         )
-        with warnings.catch_warnings(record=True) as w:
+        with warnings.catch_warnings(record=True) as _:
             warnings.simplefilter("always")
             self.project.import_from_path(path=folder_path, recursive=False)
-            self.assertEqual(len(w), 4)
```

Comment:
Is it necessary to remove this line? While I agree that counting the number of warnings is not very intuitive it helped to identify the recent HDF5 warnings, so I would prefer to keep it. In any case please make sure to merge the master branch ones as I have the feeling this pull request started before the recent HDF5 version update which might be the reason why you removed this part.

Reply:
> While I agree that counting the number of warnings is not very intuitive...

Indeed! If we keep it, it would be nice to have something in the `msg` kwarg explaining what warnings we're expecting to see here.

Reply:
I removed it precisely because it isn't that intuitive. I like Liam's idea of checking the message but I don't know what to expect and maybe this needs to be done in a separate PR.